lua/nvim-tree/utils.lua

```diff
@@ -245,4 +245,10 @@ function M.table_create_missing(tbl, sub)
   return t
 end
 
+-- @param value string
+-- @return string
+function M.trim_string(value)
+  return (string.gsub(value, "^%s*(.-)%s*$", "%1"))
+end
```

Comment:
uneeded, there is `vim.trim` function already :)

Reply:
Ah, perfect. Searched for a similar function in native lua...